client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -68,6 +69,19 @@ const CheckoutThankYouHeader = React.createClass( {
 			} );
 		}
 
+		if ( isGuidedTransfer( this.props.primaryPurchase ) ) {
+			if ( typeof this.props.primaryPurchase.meta === 'string' ) {
```

Comment:
I've done this just to be more explicit and to give us a bit more flexibility on the server-side since we haven't yet decided what the `meta` property should contain. For example, it could instead be a `siteID` or even an object, so ensuring it's a string would avoid weirdness in either of those cases (eg, "The guided transfer for [object Object] will begin soon")